quiz.js

```diff
@@ -0,0 +1,58 @@
+
+var tree = {
+	height: document.getElementById("height"),
+	character: document.getElementById("character")
+};
+
+var submitButton = document.getElementById("submit");
+var clearButton = document.getElementById("clear");
+var output = document.getElementById("treeDOM");
+
+
+function growYourTree () {
```

Comment:
since you are using a form tag change this line to:
function growYourTree (e) {